### DISCUSSION TITLE
how can I resolve parsing_exception error or transport error

### DISCUSSION THREAD

**Discussion by VivekGupta97** [apache/superset#18370] [Q&A / Help]
-1

My schema structure in elasticsearch like :- "hits" : { "total" : { "value" : 2, "relation" : "eq" }, "max_score" : 1.0, "hits" : [ { "_index" : "decdrop_repo", "_type" : "_doc", "_id" : "4172", "_score" : 1.0, } ] } .

I want to retreive _id from this schema and my query is :- SELECT 'hits.hits._id' FROM "decdrop_repo"; here I found only hits.hits._id as same but not found value of _id.

And if I query straight forward like sql then I get an errors like :- TransportError(500, 'ql_illegal_argument_exception', 'Arrays ('not supported'). or Parsing_exception error.

could anyone please suggest me how can I resolved this ? Thanks !

**Comment by geido** [comment]
Hello @VivekGupta97 we need more info to help you out. Please provide screenshots and stack traces.

**Comment by VivekGupta97** [comment]
![image](https://user-images.githubusercontent.com/68772150/149452920-3a0e07e0-9887-40c1-8561-4c60ae47657a.png)

**Comment by VivekGupta97** [comment]
I Have used superset SQL editor for Query and my database is elasticsearch and my data is in above format

**Comment by VivekGupta97** [comment]
![image](https://user-images.githubusercontent.com/68772150/149453570-1c098630-5935-4891-b2b8-d0e0d0c419f9.png)
if I run query like this then got error as
![image](https://user-images.githubusercontent.com/68772150/149453709-9868f772-b709-472c-bf49-948ecb8a4e7f.png)

**Comment by VivekGupta97** [comment]
> Hello @VivekGupta97 we need more info to help you out. Please provide screenshots and stack traces.

I hope These detail help you to understand my issue

**Comment by VivekGupta97** [comment]
Hello @geido Please help me to solve this issue how can I get the values from query in elastic search ?
I hope my issue understood by you
